UnitsNet/UnitsNet.Net35.csproj

```diff
@@ -40,7 +40,158 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs;" Exclude="obj\**;**\*.WindowsRuntimeComponent.cs" />
+    <Compile Include="AmbiguousUnitParseException.cs" />
```

Comment:
It happens whenever you add a file to the project in Visual Studio, as VS is not aware of how to work with wildcards. I believe VS2017 will get support for this. One of the steps of the wiki of adding a new unit is to close and reopen the solution after generating new source code in order to see the new files.